### PR TITLE
Set python version in mod_wsgi-ge.conf

### LINKED ISSUE #17: Publish module fails to run due to missing dependency in RHEL 7
After doing a build and deploy using Andrei’s script, the Apache Publish module does not work. (So far I have only seen this on RHEL 7) This error in the GEE error log (/opt/google/gehttpd/logs/error_log):
```
[Thu Mar 23 20:04:02 2017] [error] [client 10.240.0.3]   File "/opt/google/gehttpd/wsgi-bin/serve/publish/publish_app.wsgi", line 1, in <module>
...
[Thu Mar 23 20:04:02 2017] [error] [client 10.240.0.3]     import psycopg2  # NO LINT. third party package should be before own ones.
[Thu Mar 23 20:04:02 2017] [error] [client 10.240.0.3] ImportError: No module named psycopg2
```

### PR DESCRIPTION
**Pull request by tst-lsavoie**
fixes #17

mod_wsgi-ge.conf includes the path to the Python site-packages
file, and the path includes the Python version.  This commit
ensures that the Python version, and thus the path, is correct
in mod_wsgi-ge.conf.

### CODE REVIEW

File: earth_enterprise/src/third_party/mod_wsgi/SConscript [hunk 1]
```diff
@@ -125,6 +125,12 @@ mod_wsgi_env.ExecuteOnClean('rm -rf %s' % current_dir)
 
 if 'install' in COMMAND_LINE_TARGETS:
   server_root = mod_wsgi_env.installdirs['server_root']
+
+  # Configure mod_wsgi-ge.conf with the correct python version
+  mod_wsgi_conf = 'conf.d/mod_wsgi-ge.conf'
+  mod_wsgi_env.Command(mod_wsgi_conf, mod_wsgi_conf + '.in',
+      "sed 's/@PYTHON_VERSION@/%s/' < $SOURCE > $TARGET" % mod_wsgi_env['python_version'])
+
```
Comment (andreisel) [inline earth_enterprise/src/third_party/mod_wsgi/SConscript]: I like adding of suffix ".in" for source file, but will prefer do not create output file in src directory.

The "install"-target is being used to copy all the files from bin_dir/.../install directories to specified install_dir to prepare them for deployment/packaging.
We need to move altering of mod_wsgi-ge.conf to the creating master installer target which prepares install-files in bin/../install dir (see line 103, command after mod_wsgi_target = '%s/.install' % current_dir)
In command on line 103, you may do something like:

mkdir -p %s/gehttpd/conf.d' % install_root_opt
install -m755 SOURCE_PATH TARGET_PATH
"sed -i -e 's/@PYTHON_VERSION@/%s/' TARGET_PATH" % mod_wsgi_env['python_version']

Source, target paths something like:
mod_wsgi_dir = Dir('#third_party/mod_wsgi').abspath
mod_wsgi_conf = 'conf.d/mod_wsgi-ge.conf'
SOURCE_PATH = path.join(mod_wsgi_dir, '%s.in' % mod_wsgi_conf)
TARGET_PATH = "%s/gehttpd/conf.d/%s' % (install_root_opt, mod_wsgi_conf)

File: earth_enterprise/src/third_party/mod_wsgi/SConscript [hunk 2]
```diff
@@ -125,6 +125,12 @@ mod_wsgi_env.ExecuteOnClean('rm -rf %s' % current_dir)
...
   mod_wsgi_env.install('httpdconf', ['conf.d/mod_wsgi-ge.conf'])
```
Comment (andreisel) [inline earth_enterprise/src/third_party/mod_wsgi/SConscript]: After preparing mod_wsgi-ge.conf in "create master installer", we don't need to install mod_wsgi-ge.conf.
Please, delete line 134.

File: earth_enterprise/src/third_party/mod_wsgi/conf.d/mod_wsgi-ge.conf.in [hunk 3]
```diff
@@ -7,7 +7,7 @@ LoadModule wsgi_module modules/mod_wsgi.so
     WSGIPythonHome /usr
 
     # Handles Fusion requests.
-    WSGIDaemonProcess ge_fusion_serve threads=15 display-name=%{GROUP} python-path=/opt/google/gepython/Python-2.7.6/lib/python2.7/site-packages:/opt/google/gehttpd/wsgi-bin home=/tmp
+    WSGIDaemonProcess ge_fusion_serve threads=15 display-name=%{GROUP} python-path=/opt/google/gepython/Python-@PYTHON_VERSION@/lib/python2.7/site-packages:/opt/google/gehttpd/wsgi-bin home=/tmp
```
Comment (andreisel) [inline earth_enterprise/src/third_party/mod_wsgi/conf.d/mod_wsgi-ge.conf.in]: We could also replace '2.7' (use mod_wsgi_env['python_version']) in .. lib/python2.7.

**Comment by tst-lsavoie** [comment]
This tests out on RHEL 7 and Ubuntu 14.04 and has two approvals, so I think it's ready to merge.  I don't have write access, so someone else will have to do that.

**Comment by tst-ccamp** [comment]
Squashing and Merging

**Pull request merged**